litmus-portal/frontend/cypress/component/passedVsFailed.spec.tsx

```diff
@@ -14,20 +14,6 @@ describe('Passed Vs Failed: props -> passed: 75, failed: 25', () => {
     expect(wrapper.props.failed).to.equal(25);
   });
 
-  it('Pass Icon is present', () => {
-    mount(wrapper);
-    cy.get('[data-cy=passIcon]')
-      .should('have.attr', 'src')
```

Comment:
did we plan on removing all test cases @Jonsy13 @S-ayanide

Reply:
yeah we are not using these icons i am using the available icons in material ui so these conditions will always fail @arkajyotiMukherjee cc: @Jonsy13 @S-ayanide

Reply:
No @arkajyotiMukherjee It's not decided.Tests will remain.

Reply:
without icons how do i make the old test cases pass? @Jonsy13 @arkajyotiMukherjee

Reply:
@ishangupta-ds Can we add the two tests but instead use you Avatar icon and other icons from MUI itself? If that's possible for now!

Reply:
Yes @S-ayanide That should be good.

Reply:
It is not accessible as png now we have to check for the material icon, i have another PR pending for the analytics, will be adding test cases after this release @S-ayanide @Jonsy13 ..if it is good?